Currency.py

```diff
@@ -50,34 +52,41 @@ def convert(self, amount, src, dst):
             dst_rate = self.exchange_rates[dst]
             return str(amount / src_rate * dst_rate)
 
-class Yahoo:
+class Api:
```

Comment:
please give this a reasonable name an reintroduce yahoo.

Reply:
Can do. Yahoo does not work for me. I tested the regex on the html that was returned to me and it was redirected to Yahoo AU. So there may be inconsistencies in results across regions and the Yahoo Finance API has been deprecated for a while.
![screenshot_2018-12-24_16-48-30](https://user-images.githubusercontent.com/44654729/50391763-8f2e9c80-079c-11e9-8c6a-70ab97d197e5.png)